datastore/statedata.go

```diff
@@ -269,6 +270,9 @@ type UpdateInfo struct {
 	// data and discover that it is a different version. See also the
 	// StateDataKeyUncommitted key.
 	HasDBSchemaUpdate bool
+
+	// UpdateControl map from the server FIXME (write proper)
+	UpdateControlMap interface{} `json:"update_control_map"` // TODO - should this be the updatemanager control map (?)
```

Comment:
As mentioned in an earlier comment, I don't think we need to store this.

Reply:
Yeh, agree. Removed.